### PR TITLE
add WME info

### PR DESCRIPTION
**Pull request by rosemarymarano**

### CODE REVIEW

**Review by mriccell — APPROVED** [review]

File: documentation/staging/content/userguide/managing-domains/_index.md [hunk 1]
```diff
@@ -153,6 +154,13 @@ The operator lets you initiate scaling of clusters in various ways:
 The operator generates Kubernetes events at key points during domain processing.
 For more information, see [Domain events]({{< relref "/userguide/managing-domains/domain-events.md" >}}).
 
+### Monitoring a domain
+
+Use the WebLogic Monitoring Exporter, a [Prometheus](https://prometheus.io/)-compatible exporter of metrics from WebLogic Server instances, which it obtains by using the [WLS RESTful Management API](https://docs.oracle.com/en/middleware/fusion-middleware/weblogic-server/12.2.1.4/wlrur/overview.html#GUID-B193E8EF-1912-48D1-8FB9-99C5ADACCC3B), available in version 12.2.1 or later. Metrics are selected using a [YAML configuration file](https://github.com/oracle/weblogic-monitoring-exporter/blob/89796bba357642e51387ef1bc17a2626f29d0b2f/README.md#Configuration).
```
Comment (russgold) [inline documentation/staging/content/userguide/managing-domains/_index.md:159]: If you follow this link (and the next one), you'll find it doesn't go where you expect it to. 

I think that the description here needs to be more operator-centric, and less focused on the details of the WME. The only link should probably be to that project.

How do we usually describe domain-related features?

**Review by russgold — CHANGES_REQUESTED** [review]

File: documentation/staging/content/userguide/managing-domains/_index.md (hunk 1 above)
Comment (rosemarymarano) [reply documentation/staging/content/userguide/managing-domains/_index.md:159]: @russgold "If you follow this link (and the next one), you'll find it doesn't go where you expect it to." whether I copy and paste this link or view it in Hugo, it goes to the WME Configuration section, example YAML file configuration. Is that wrong? The next one,  a [separate process](https://github.com/oracle/weblogic-monitoring-exporter/blob/89796bba357642e51387ef1bc17a2626f29d0b2f/README.md#sidecar) goes to the Sidecar section in the WME docs. Is that wrong?

Comment (rosemarymarano) [reply documentation/staging/content/userguide/managing-domains/_index.md:159]: @russgold "I think that the description here needs to be more operator-centric, and less focused on the details of the WME. The only link should probably be to that project. How do we usually describe domain-related features" It seems to me that the topics on this page, [Manage WebLogic domains](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/) describes briefly domain-related features, then points off to sections where they are described in more detail, if such sections exist. WE don't have much of anything in the User Guide for domain monitoring or logging and [this](https://oracle.github.io/weblogic-kubernetes-operator/samples/elastic-stack/weblogic-domain/) in the Samples guide. If the WKO docs are lacking certain content, perhaps you or @mriccell can create a JIRA for it.

Comment (russgold) [reply documentation/staging/content/userguide/managing-domains/_index.md:159]: > @russgold "If you follow this link (and the next one), you'll find it doesn't go where you expect it to." whether I copy and paste this link or view it in Hugo, it goes to the WME Configuration section, example YAML file configuration. Is that wrong? The next one, a [separate process](https://github.com/oracle/weblogic-monitoring-exporter/blob/89796bba357642e51387ef1bc17a2626f29d0b2f/README.md#sidecar) goes to the Sidecar section in the WME docs. Is that wrong?

Yes, it is going to a version that is not in a branch or main. If you scroll up, you'll see that a comment telling you that, and it still has the link to the test sample file.

Comment (rosemarymarano) [reply documentation/staging/content/userguide/managing-domains/_index.md:159]: @russgold OH, now I understand. I'll fix it now that the WME PR is merged.

Comment (russgold) [reply documentation/staging/content/userguide/managing-domains/_index.md:159]: > @russgold "I think that the description here needs to be more operator-centric, and less focused on the details of the WME. The only link should probably be to that project. How do we usually describe domain-related features?" It seems to me that the topics on this page, [Manage WebLogic domains](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/) describe briefly domain-related features, then point off to sections where they are described in more detail, if such sections exist. We don't have much of anything in the User Guide for domain monitoring or logging and [this](https://oracle.github.io/weblogic-kubernetes-operator/samples/elastic-stack/weblogic-domain/) in the Samples guide. If the WKO docs are lacking certain content, perhaps you or @mriccell can create a JIRA for it.

I see the link to the Domain page, which describes various elements in the domain spec. That is what I was thinking of.

But if the goal is simply a high-level reference, I'd say that there is too much, here, and propose something like:

"The operator can export Prometheus-compatible metrics by embedding a WME-configuration in its domain specification. See the WME doc https://github.com/oracle/weblogic-monitoring-exporter#use-the-monitoring-exporter-with-weblogic-kubernetes-operator" for more details.

Comment (rosemarymarano) [reply documentation/staging/content/userguide/managing-domains/_index.md:159]: @russgold @mriccell I agree. They can get the WME details from the WME README.

**Review by mriccell — APPROVED** [review]

**Review by russgold — APPROVED** [review]

File: documentation/staging/content/userguide/managing-domains/_index.md (hunk 1 above)
Comment (mriccell) [reply documentation/staging/content/userguide/managing-domains/_index.md]: agreed

**Pull request merged**